net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -191,6 +191,12 @@ mwan3_init()
 	MMX_DEFAULT=$(mwan3_id2mask mmdefault MMX_MASK)
 	MMX_BLACKHOLE=$(mwan3_id2mask MM_BLACKHOLE MMX_MASK)
 	MMX_UNREACHABLE=$(mwan3_id2mask MM_UNREACHABLE MMX_MASK)
+
+	# The workaround is only needed for older kernel version.
+	# See issue: https://bugs.openwrt.org/index.php?do=details&task_id=2897
+	# See issue: https://github.com/openwrt/packages/issues/13655
+	KERNEL_HAS_SK_BUG=false
+	opkg compare-versions $(uname -r) '>' '4.14.209' || KERNEL_HAS_SK_BUG=true
```

Comment:
I would put a warning in syslog suggesting the upgrade each time mwan3 is restarted. It would not be too much chatty.